tests/migration_test.go

```diff
@@ -4451,6 +4453,53 @@ var _ = Describe("[rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][level:system
 		})
 
 	})
+
+	Context("when migrating fails", func() {
+		var vmi *v1.VirtualMachineInstance
+
+		BeforeEach(func() {
+			vmi = libvmi.NewCirros(libvmi.WithMasqueradeNetworking()...)
+			vmi.Annotations = map[string]string{v1.FuncTestForceLauncherMigrationFailureAnnotation: ""}
```

Comment:
There is also the option for this, could you use it?

Reply:
yup, thanks.